### DISCUSSION TITLE
Slider number stays for longer if missed

### DISCUSSION THREAD

**Discussion by agentceci** [ppy/osu#19795] [Q&A] [answered]
### Type

Game behaviour

### Bug description

The number on the slider in lazer seems to stay for longer if I miss the slider completely.
Issue is found across all standard mods except hidden.

### Screenshots or videos

https://streamable.com/bm9zd3 here I am comparing it to stable's slider miss

### Version

2022.810.2-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/9339697/network.log)
[performance.log](https://github.com/ppy/osu/files/9339698/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9339700/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9339701/updater.log)
[database.log](https://github.com/ppy/osu/files/9339709/database.log)

**Comment by peppy** [comment]
The linked video is showing stable gameplay. I'm not really sure what it's trying to show.

Someone will need to elaborate on this.

**Reply by agentceci** [reply]
halfway through i switch to lazer gameplay (0:00-0:16-stable, 0:17-0:35-lazer / when background dim changes)

**Comment by agentceci** [comment]
I made another video which displays the issue more clearly, I hope - https://streamable.com/4tuzzd

**Comment by JEllermann** [comment]
Old post, but I can also observe the problem. Since this only happens sometimes (I think only if you don't hit 300), it feels like a lag and is very distracting. You can only notice it if you use a skin that does not use a SliderBall or a transparent one. Therefore, it is not noticeable with the standard skins except you are not hitting the slider at all.

**Reply by Flamiii** [reply, marked as answer]
The behavior that you're describing is actually an intended game mechanic and is not a bug. It's known as "slider head movement" and it can be disabled through the use of the Classic mod if you prefer the old behavior of osu!stable. 

There is ongoing discussion regarding whether or not this should be a game mechanic in other github threads, such as this one: https://github.com/ppy/osu/discussions/23320
